tests/unit/legate/tester/test_args.py

```diff
@@ -66,6 +68,18 @@ def test_ompthreads(self) -> None:
     def test_numamem(self) -> None:
         assert m.parser.get_default("numamem") == DEFAULT_NUMAMEM
 
+    def test_ranks(self) -> None:
+        assert m.parser.get_default("ranks") == DEFAULT_RANKS_PER_NODE
```

Comment:
FYI @manopapad recently renamed this to `ranks_per_node` in the driver, maybe we should change here to match?